openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html

```diff
@@ -22,14 +22,14 @@
                         <label for="peer_assessment_must_grade" class="setting-label">{% trans "Must Grade" %}</label>
                         <input id="peer_assessment_must_grade" class="input setting-input" type="number" value="{{ assessments.peer_assessment.must_grade }}" min="0" max="99">
                     </div>
-                    <p class="setting-help">{% trans "Specify the number of peer assessments that each student must complete."%}</p>
+                    <p class="setting-help">{% trans "Specify the number of peer assessments that each student must complete. Maximum: 99"%}</p>
```

Comment:
"Specify the number of peer assessments that each learner must complete. Valid numbers are 0 to 99."